### PR TITLE
Feat: adding the new prop openOnIconOnly

### PR DESCRIPTION
**Pull request by gabrielforster**
### Description
Adding new prop to open the component only when clicking on the icon

### CODE REVIEW

**Review by VictorTavaress — APPROVED** [review]
LGBT

File: src/components/DataVisualization/Accordion/Accordion.vue [hunk 1]
```diff
@@ -71,25 +71,33 @@ export default {
...
+      type: Boolean,
+      default: false,
+    },
   },
 
   data() {
     return {
       state: {
         collapsed: this.open,
+        test: false,
```
Comment (aXhwZA) [inline src/components/DataVisualization/Accordion/Accordion.vue:84]:
```suggestion
```

**Review by aXhwZA — CHANGES_REQUESTED** [review]

**Review by VictorTavaress — APPROVED** [review]
LGTM<

**Review by aXhwZA — APPROVED** [review]
LGTM

**Pull request merged**